client/bindings/nodejs/types/preparedTransactionData.ts

```diff
@@ -65,5 +65,5 @@ export interface IRemainder {
 }
 export interface ISegment {
     hardened: boolean;
-    bs: Uint8Array;
+    bs: number[];
```

Comment:
Maybe elaborate the changelog a tiny bit more

Reply:
Please check now